src/tv2-common/onTimelineGenerate.ts

```diff
@@ -88,16 +89,19 @@ export function onTimelineGenerate<
 ): Promise<BlueprintResultTimeline> {
 	const previousPartEndState2 = previousPartEndState as PartEndStateExt | undefined
 	const persistentState: TimelinePersistentStateExt = {
-		activeMediaPlayers: {}
+		activeMediaPlayers: {},
+		isNewSegment: context.previousPartInstance?.segmentId !== context.currentPartInstance?.segmentId
```

Comment:
Yes, it is, but we were in a wierd place where I needed the SegmentId of the Part we are cutting to and of the Part we are cutting from. 
When the 'previousPartEndState' is calculated it only knows about the SegmentId of the Part we are cutting from and of the Part before the Part we are cutting from. So we didn't have the necessary information.

I decided it wasn't too bad to add 'previousPartInstance' to the context since it already was in the context, it just wasn't exposed.

Reply:
I don't think that works fine in this scenario: Segment B has a Live 1 (wanting to persist) as its first part. 
1. Take any part in Segment A
2. set Segment B as next
3. Take
4. cut to Kam 1 (accepting persistence)
The levels from Live 1 are not persisted